src/javascripts/helpers/login_helper.js

```diff
@@ -1,19 +1,28 @@
-var password = document.getElementById('input-password');
-var username = document.getElementById('input-username');
-var agentPw = 'travel2020';
-var agentUn = 'agency';
 var button = document.getElementById('submit');
+var passwordInput = document.getElementById('input-password');
+var username = document.getElementById('input-username');
+var password = "travel2020";
+var agency = "agency";
 
-function authPW() {
-  if (password.value == agentPw && username.value == agentUn) {
+function authPassAgent() {
```

Comment:
I think the best thing to do here would be to split this login logic up in to smaller chunks of functionality

```
function isValidPassword(passwordInput) {
// check the password === passwordInput
}

function isValidTraveler(usernameInput) {
   travelers = getTravelers();  // this will call a method in the travel_tracker_service
  usernameInput == a  valid traveler
}

function isValidAgency(usernameInput) {
   usernameInput == agency
}

function isValidUser(usenamerInput) {
    isValidAgency(usernameInput) || isValidUser(usernameInput)
}

function loginSuccessful(usernameInput, passwordInput) {
    isValidUser(usernameInput) && isValidPassword
}

function renderSuccessfulAgentLogin () {
//spit out html for agent login
}

// function do the same for traveler login

// then the below function could be used for the form submission "onClick"

function processLogin(usernameInput, passwordInput) {
    if (loginSuccessful(usernameInput, passwordInput)){
       if (isValidTravelerr(usernameInput)) {
         renderSuccessfulTravlerLogin;
      } else if (isValidAgency(usernameInput)) {
        renderValidAgencyLogin;
      } 
   }  else {
     alert("Login Unsuccessful!")
   }
}
```